web/src/engine/websites/MundoMangaKun_e2e.ts

```diff
@@ -0,0 +1,26 @@
+﻿import { TestFixture, type Config } from '../../../test/WebsitesFixture';
+
+const config: Config = {
+    plugin: {
+        id: 'mundomangakun',
+        title: 'Mundo Mangá-Kun'
+    },
+    container: {
+        url: 'https://mundomangakun.com.br/manga/a-perverts-daily-life/',
+        id: '/manga/a-perverts-daily-life/',
+        title: 'A Pervert’s Daily Life'
+    },
+    child: {
+        id: '/a-perverts-daily-life-capitulo/',
+        title: 'Capítulo 01',
+        timeout : 23000
```

Comment:
Maybe consider using a chapter that has less images in hope it loads fatser.

Reply:
Yup i knew you'd said that. I tried other chapters before this one it was worst.